data/human/jobs.txt

```diff
@@ -3781,3 +3781,110 @@ mission "Hunted"
 		system
 			distance 5 5
 		fleet "Marauder fleet X"
+
+
+mission "Republic Navy Advisory System [frozen]"
+	name "Cold Relief"
+	description "Humanitarian aid of <cargo> to <destination> by <date> following a severe cold snap."
+	minor
+	repeat
+	deadline
+	source
+		government "Republic"
```

Comment:
```suggestion
		government "Republic" "Syndicate"
```
Syndicate planets are part of the Republic too and get Navy protection, so they'd presumably help out.